packages/mock-account-provider/app/lib/requesters.ts

```diff
@@ -152,6 +154,43 @@ export async function createPaymentPointer(
     })
 }
 
+export async function createPaymentPointerKey({
+  paymentPointerId,
+  jwk
+}: {
+  paymentPointerId: string
+  jwk: string
+}): Promise<CreatePaymentPointerKeyMutationResponse> {
+  const createPaymentPointerKeyMutation = gql`
+    mutation CreatePaymentPointerKey($input: CreatePaymentPointerKeyInput!) {
+      createPaymentPointerKey(input: $input) {
+        code
+        success
+        message
+      }
+    }
+  `
+  const createPaymentPointerKeyInput: CreatePaymentPointerKeyInput = {
+    paymentPointerId,
+    jwk
```

Comment:
I like the idea of making `jwk` in this mutation input not a string, but explicitly:
```js
{
    alg,
    kid,
    kty,
    crv,
    x
  }
```

It feels a little too loose with a string, given Open Payment validation will fail if asking for `{{paymentPointer}}/jwks.json`, and we don't have all of those fields